test/transaction/vote_cr_tx.lua

```diff
@@ -43,24 +41,22 @@ then
     fee = 0.1
 end
 
-if vote_candidate_vote == 0
+if vote_candidates == ""
```

Comment:
should be {}

Reply:
should be if next(vote_candidates) == nil